src/server/auth/cmds/cmds.go

```diff
@@ -194,6 +179,14 @@ func LoginCmd() *cobra.Command {
 				resp, authErr = c.Authenticate(
 					c.Ctx(),
 					&auth.AuthenticateRequest{OneTimePassword: code})
+			} else if useToken {
```

Comment:
This is identical to how the OTP code works, but it just stuffs a token into the pachd config directly.

Reply:
Yup, I just didn't know we had a use-auth-token. I'll remove this for now.